tests/python/python_structure_manager_test.py

```diff
@@ -227,7 +228,7 @@ def test_mask_id_select(self):
         mask_center_atoms_by_id(self.frame, np.arange(3, 7))
         test_mask[:7] = True
         self.check_mask(test_mask)
-        mask_center_atoms_by_id(self.frame, id_blacklist=[0,])
+        mask_center_atoms_by_id(self.frame, id_blacklist=[0, ])
```

Comment:
unrelated, by why is there a comma here and below? this is not needed for lists. Ping @max-veit

Reply:
it is not needed but not a problem either no ?